content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -149,6 +155,56 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+> Note that unlike `.dvc` files, `dvc.yaml` doesn't store any `md5` hash values,
```

Comment:
let's remove this note ... people don't expect those md5 to be stored in the dvc.yaml in the first place (only user who had experience with DVC < 1.0)

Reply:
Removed, but I think since there's `md5` in .dvc files, some users may expect them in dvc.yaml